ood-gen.opam

```diff
@@ -10,7 +10,7 @@ bug-reports: "https://github.com/patricoferris/ood/issues"
 depends: [
   "dune" {>= "2.8"}
   "ocaml" {>= "4.08.0"}
-  "yaml"
+  "yaml" {>= "3.0"}
```

Comment:
Why the addition of the lower bound? Just curious.

Reply:
Nvm, that's explained in the issue description.

Reply:
Mainly for https://github.com/avsm/ocaml-yaml/pull/47